package.json

```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "@daostack/arc": "0.0.0-alpha.56",
-    "@graphprotocol/graph-cli": "./graphprotocol-graph-cli-0.4.0.tgz",
+    "@graphprotocol/graph-cli": "^0.4.0",
```

Comment:
this patch is here because https://github.com/graphprotocol/graph-cli/pull/165 which is not yet released.